drupal/sites/all/features/boinc_standard/boinc_standard.features.menu_links.inc

```diff
@@ -82,11 +82,10 @@ function boinc_standard_menu_default_menu_links() {
   );
   // Translatables
   // Included for use with string extractors like potx.
```

Comment:
I think it would be best to remove them since they are inaccurate now. The comment suggests that you could remove the following code because it is only needed for potx extraction which is wrong. It is needed for normal operation now that we use `bts()` and context information.